www/base/src/app/settings/service/settings.service.coffee

```diff
@@ -0,0 +1,45 @@
+class bbSettings extends Provider
```

Comment:
That magic is actually ng-classify magic. See here : https://github.com/CaryLandholt/ng-classify#provider

But in Guanlecoja, the default renaming of 'blabla' in 'blablaProvider' is actually 'blablaServiceProvider', so all the services are named and accessed with 'blablaService'.